test/smtcallback.ts

```diff
@@ -25,7 +25,7 @@ function collectErrors (solOutput) {
   return errors;
 }
 
-function expectErrors (expectations, errors, ignoreCex) {
+function expectErrors (expectations, errors, ignoreCex: any) {
```

Comment:
`ignoreCex` is now required, I guess we do not need the type for now?

Reply:
yeah its no longer needed, if its required and used then the `any` flag can be removed and or type defined.